client/state/themes/actions.js

```diff
@@ -689,11 +689,11 @@ export function deleteTheme( themeId, siteId ) {
 export function confirmDelete( themeId, siteId ) {
 	return ( dispatch, getState ) => {
 		const { name: themeName } = getTheme( getState(), siteId, themeId );
-		const siteSlug = getSiteSlug( getState(), siteId );
+		const siteTitle = getSiteTitle( getState(), siteId );
 		accept(
 			i18n.translate(
-				'Are you sure you want to delete %(themeName)s from %(siteSlug)s?',
-				{ args: { themeName, siteSlug }, context: 'Themes: theme delete confirmation dialog' }
+				'Are you sure you want to delete %(themeName)s from %(siteTitle)s?',
```

Comment:
@yoavf We've changed the argument name here, so the bot shouldn't suggest to keep the old string (with the old arg).